### DISCUSSION TITLE
Why was github actions down for 10 hours?

### DISCUSSION THREAD

**Discussion by david-gang** [community/community#27103] [Actions]
<p>Hi,</p>
<p>There was an incident on the 18.05 which took github actions down for 10 hours: <a href="https://www.githubstatus.com/incidents/m16jzl31gnqt" class="inline-onebox" rel="noopener nofollow ugc">GitHub Status - Incident with GitHub Actions</a></p>
<p>Normally when working with google cloud, they provide a short post about the root cause of the issue. For example here: <a href="https://status.cloud.google.com/incidents/aVUuw5cr7iMM6UbuwyYb" class="inline-onebox" rel="noopener nofollow ugc">Google Cloud Status Dashboard</a></p>
<p>Here in github actions i did not see a post about the root cause. Does anyone know what the root cause was? This would help the customers trust.</p>
<p>Thanks</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi <a class="mention" href="/david-gang">@david-gang</a> :wave: Apologies for the delay here.</p>
<p>I have added your feedback to an internal tracking issue discussing how we document incidents.</p>
<p>At present, you can certainly read more here:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">
      <a href="https://github.blog/2021-06-02-github-availability-report-may-2021/" target="_blank" rel="noopener" title="12:00AM - 03 June 2021">The GitHub Blog – 3 Jun 21</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://github.blog/wp-content/uploads/2021/04/Blog_ENGINEERING_for-social.png?fit=1200%2C630" class="thumbnail" width="690" height="362"></div>

<h3><a href="https://github.blog/2021-06-02-github-availability-report-may-2021/" target="_blank" rel="noopener">GitHub Availability Report: May 2021 | The GitHub Blog</a></h3>

<p>In May, we experienced two incidents resulting in significant impact to multiple GitHub services.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>And if you have concerns, you can of course post them here or <a href="https://support.github.com/contact">open a ticket</a> with support.</p>
